gnomad/utils/vcf.py

```diff
@@ -14,6 +112,160 @@
+    },
+    "MQRankSum": {
+        "Description": "Z-score from Wilcoxon rank sum test of alternate vs. reference read mapping qualities"
+    },
+    "QD": {
+        "Description": "Variant call confidence normalized by depth of sample reads supporting a variant"
+    },
+    "ReadPosRankSum": {
+        "Description": "Z-score from Wilcoxon rank sum test of alternate vs. reference read position bias"
+    },
+    "SOR": {"Description": "Strand bias estimated by the symmetric odds ratio test"},
+    "VQSR_POSITIVE_TRAIN_SITE": {
+        "Description": "Variant was used to build the positive training set of high-quality variants for VQSR"
+    },
+    "VQSR_NEGATIVE_TRAIN_SITE": {
+        "Description": "Variant was used to build the negative training set of low-quality variants for VQSR"
+    },
+    "BaseQRankSum": {
+        "Description": "Z-score from Wilcoxon rank sum test of alternate vs. reference base qualities",
+    },
+    "VarDP": {
+        "Description": "Depth over variant genotypes (does not include depth of reference samples)"
+    },
+    "AS_VQSLOD": {
+        "Number": "A",
+        "Description": "Allele-specific log-odds ratio of being a true variant versus being a false positive under the trained VQSR Gaussian mixture model",
+    },
+    "AS_VQSR_culprit": {
+        "Number": "A",
+        "Description": "Allele-specific worst-performing annotation in the VQSR Gaussian mixture model",
+    },
+    "decoy": {"Description": "Variant falls within a reference decoy region"},
+    "lcr": {"Description": "Variant falls within a low complexity region"},
+    "nonpar": {
+        "Description": "Variant (on sex chromosome) falls outside a pseudoautosomal region"
+    },
+    "segdup": {"Description": "Variant falls within a segmental duplication region"},
+    "rf_positive_label": {
+        "Description": "Variant was labelled as a positive example for training of random forest model"
+    },
+    "rf_negative_label": {
+        "Description": "Variant was labelled as a negative example for training of random forest model"
+    },
+    "rf_label": {"Description": "Random forest training label"},
+    "rf_train": {"Description": "Variant was used in training random forest model"},
+    "rf_tp_probability": {
+        "Description": "Probability of a called variant being a true variant as determined by random forest model"
+    },
+    "transmitted_singleton": {
+        "Description": "Variant was a callset-wide doubleton that was transmitted within a family from a parent to a child (i.e., a singleton amongst unrelated samples in cohort)"
+    },
+    "original_alleles": {"Description": "Alleles before splitting multiallelics"},
+    "variant_type": {
+        "Description": "Variant type (snv, indel, multi-snv, multi-indel, or mixed)"
+    },
+    "allele_type": {
+        "Number": "A",
+        "Description": "Allele type (snv, insertion, deletion, or mixed)",
```

Comment:
good point. I took this straight out of the gnomAD release code. I'll remove the "Number": "A" since this is a site annotation (we add it with https://github.com/broadinstitute/gnomad_methods/blob/e78dea469af3d90097edbe2d9d763216d874ee4b/gnomad/utils/annotations.py#L716 before splitting multiallelics)

Reply:
good point, it is really more of a site_type. @gtiao, do you think we should we change it at this point, or keep as is to be consistent with past call sets? I lean to keep, but fine either way.